docs/packages/pkg/aws-c-common.rst

```diff
@@ -0,0 +1,22 @@
+.. spelling::
+
+    aws
+    aws-c-common
+
+.. index::
+  single: aws ; aws-c-common
+
+.. _pkg.aws-c-common:
+
+aws-c-common
+============
+
+-  `Official <https://github.com/awslabs/aws-c-common>`__
+-  `Hunterized <https://github.com/hunter-packages/aws-c-common>`__
```

Comment:
There is no such repository. Let me know if you need it.

Reply:
Please fork my repository or the original repository, in case I delete my fork or account.

Reply:
> Please fork my repository or the original repository, in case I delete my fork or account

Done: https://github.com/hunter-packages/aws-c-common Branch `hunter-0.2.1` created from upstream tag `v0.2.1`. Please also read https://docs.hunter.sh/en/latest/creating-new/patch.html Note that in general it's better to ask for a fork before submitting pull request, you can do it in issue with "New package" request.